### PR TITLE
Fix links EU

### PR DESCRIPTION
**Pull request by beatrizserrano**

### CODE REVIEW

File: content/eu/main.md [hunk 1]
```diff
@@ -19,49 +19,49 @@ subtitle: The homepage of the European Galaxy community
...
+  <div class="card border-secondary bg-light mb-1 mx-1" style="width: 18rem">
     <div class="card-body">
       <h2 class="card-title text-dark">Communities</h2>
       <p class="card-text">To address the needs of scientific & local communities, we offer customized Galaxy frontpages.</p>
       <div class="text-center">
         <img src="/images/undraw-illustrations/communities.svg" alt="communities" height="100">
-        <br><br>
-        <a href="subdomains.md" class="btn btn-primary">Check all the subdomains</a>
+        <br><br><br>
+        <a href="./subdomains.md" class="btn btn-primary">Check all the subdomains</a>
```
Comment (beatrizserrano) [inline content/eu/main.md:40]: @NickSto this is what I've mentioned in chat

File: content/eu/main.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ subtitle: The homepage of the European Galaxy community
       <div class="text-center">
         <img src="/images/undraw-illustrations/communities.svg" alt="communities" height="100">
         <br><br><br>
-        <a href="./subdomains.md" class="btn btn-primary">Check all the subdomains</a>
+        <a href="/subdomains/index.md" class="btn btn-primary">Check all the subdomains</a>
```
Comment (NickSto) [inline content/eu/main.md:40]: This needs to be just `/subdomains/`. Markdown filenames never appear in urls.

Comment (beatrizserrano) [reply content/eu/main.md:40]: hmm that doesn't find it either, I don't understand why

Comment (beatrizserrano) [reply content/eu/main.md]: oh found it, it should be `/eu/subdomains/`

**Pull request merged**